vendor/github.com/onsi/gomega/go.mod

```diff
@@ -13,3 +13,5 @@ require (
 	gopkg.in/tomb.v1 v1.0.0-20141024135613-dd632973f1e7 // indirect
 	gopkg.in/yaml.v2 v2.2.1
 )
+
+go 1.13
```

Comment:
@wzheng1 please revert this (not clear why it is here).

Reply:
Have reverted.